pkg/archive/changes_linux.go

```diff
@@ -197,7 +197,7 @@ func (w *walker) walk(path string, i1, i2 os.FileInfo) (err error) {
 	return nil
 }
 
-// {name,inode} pairs used to support the early-pruning logic of the walker type
+// {name,ino} pairs used to support the early-pruning logic of the walker type
```

Comment:
I'm -1 for this change, "ino" does not makes sense to me without description about "inode"